FFXCutsceneRemover/Components/MemoryWatchers.cs

```diff
@@ -45,6 +45,16 @@ sealed class MemoryWatchers
         public MemoryWatcher<int> HpEnemyA;
         public MemoryWatcher<byte> GuadoCount;
 
+        public MemoryWatcher<byte> EnableAuron;
+
+        public MemoryWatcher<byte> LucaFlag;
+        public MemoryWatcher<byte> LucaFlag2;
+
+        public MemoryWatcher<byte> MiihenFlag1;
+        public MemoryWatcher<byte> MiihenFlag2;
+        public MemoryWatcher<byte> MiihenFlag3;
+        public MemoryWatcher<byte> MiihenFlag4;
```

Comment:
Not sure how feasible this is but a MemoryWatcher<byte[]> array will be useful for these flags that are close to each other in memory. Also needed for sorting out formations

Reply:
Let's chat about what this implementation might look like offline.